common/azure_auth_test.go

```diff
@@ -167,7 +165,8 @@ func TestAzureAuth_ensureWorkspaceURL(t *testing.T) {
 	defer server.Close()
 
 	aa.ResourceID = "/subscriptions/a/resourceGroups/b/providers/Microsoft.Databricks/workspaces/c"
-	aa.azureManagementEndpoint = server.URL
+	// resource management endpoints end with a trailing slash in url
+	aa.azureManagementEndpoint = fmt.Sprintf("%s/", server.URL)
```

Comment:
why not simple `server.URL + "/"` ?

Reply:
noticed it after it was merged. I use had more stuff there before forgot to change it to a +.